modules/ROOT/pages/vantagecloud-lake-demos-visual-studio-code.adoc

```diff
@@ -0,0 +1,111 @@
+
+LAKE_DEMOS
+
+* https://github.com/Teradata/lake-demos/tree/main/UseCases[UseCases]
+** https://github.com/Teradata/lake-demos/blob/main/0_Demo_Environment_Setup.ipynb[0_Demo_Environment_Setup.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/1_Load_Base_Demo_Data.ipynb[1_Load_Base_Demo_Data.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Engineering_Exploration.ipynb[Data_Engineering_Exploration.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Science_OAF.ipynb[Data_Science_OAF.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Demo_Admin.ipynb[Demo_Admin.ipynb]
+* https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]
+
+Navigate to the *vars.json* file and include the following environment variables:
+
+----
+   "environment":{"host":"XXX.XXX.XXX.XXX", 
+                   "UES_URI":"", 
+                   "bucket":"", 
+                   "access_key":"", 
+                   "secret":""},    
+----
+
+Open *0_Demo_Environment_Setup.ipynb* and click on Select Kernel at the top right corner of Visual Studio Code. 
+
+If you have not installed Jupyter and Python extensions, Visual Studio Code will prompt you to install them. These extensions are necessary for Visual Studio Code to detect Kernels. To install them, select 'Install/Enable suggested extensions for Python and Jupyter.'
+
+image::select.kernel.png[select.kernel.png,align="center"]
+
+Once you've installed the necessary extensions, you'll find options in the drop-down menu. Choose **Existing Jupyter Kernel**.
+
+image::existing.kernel.png[existing.kernel.png,align="center"]
+
+Enter the URL of the running Jupyter Server and press enter.
+----
+http://localhost:8888
+----
+image::server.url.png[server.url.png,align="center"]
+
+Enter the token found in your terminal when mounting files to the Docker container and press Enter.
+
+image::server.password.png[server.password.png,align="center"]
+
+Change Server Display Name(Leave Blank To Use URL)
+
+image::server.display.name.png[server.display.name.png,align="center"]
+
+You now have access to all the Teradata Vantage extension kernels. Select Python 3(ipykernel) from the running Jupyter server.
+
+image::python.kernel.png[python.kernel.png,align="center"]
+
+You are now ready to execute all cells in *0_Demo_Environment_Setup.ipynb* to setup your environment, followed by *1_Demo_Setup_Base_Data.ipynb* to load the base data required for demo.
+
+image::demoenvsetup.png[demoenvsetup.png,align="center", width=70%]
+
+== Modify path to vars.json in UseCases directory
+
+In the UseCases directory, all .ipynb files use the path ../../vars.json to load the variables from the JSON file when working from Jupyterlab. To work directly from Visual Studio Code, update the code in each .ipynb to point to vars.json.
+
+----
+# load vars json
+with open('../../vars.json', 'r') as f:
+    session_vars = json.load(f)
```

Comment:
Let's keep it for now, but let's dive deeper in our 1:1 and see if this can be improved with VS Code configuration.